### PR TITLE
#307 Implement Umami Event Tracking

### PR DESCRIPTION
**Pull request by stephenmartin2951**

### CODE REVIEW

File: frontend/components/home/hero.tsx [hunk 1]
```diff
@@ -16,11 +16,11 @@ function Hero(): JSX.Element {
 							{t('home.hero.body')}
 						</p>
 
-						<div className="mx-auto mt-5 max-w-md sm:flex sm:justify-center md:mt-8">
-							<LinkButtonLG href="/create-review">
+						<div className="mx-auto mt-5 max-w-md sm:flex sm:justify-center md:mt-8" >
+							<LinkButtonLG href="/create-review" umami="homepage Submit a Review button"  >
```
Comment (kellenwiltshire) [inline frontend/components/home/hero.tsx:20]: Believe this needs to be data-umami-event

File: frontend/components/home/hero.tsx [hunk 2]
```diff
@@ -16,11 +16,11 @@ function Hero(): JSX.Element {
...
 								{t('home.hero.submit')}
 							</LinkButtonLG>
-							<LinkButtonLightLG href="/reviews">
+							<LinkButtonLightLG href="/reviews" umami="homepage Read Reviews button">
```
Comment (kellenwiltshire) [inline frontend/components/home/hero.tsx:23]: Same here

File: frontend/components/landlord/LandlordInfo.tsx [hunk 3]
```diff
@@ -49,8 +49,8 @@ const LandlordInfo = ({name, average, total}: IProps) => {
 				<p className="mt-1 text-sm text-gray-600">{t('landlord.share-sub')}</p>
 
 				<div>
-					<Link href="/create-review">
-						<p className="mt-2 inline-flex cursor-pointer items-center rounded-md border border-transparent bg-teal-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-teal-700 focus:outline-none focus:ring-2 focus:ring-teal-500 focus:ring-offset-2">
+					<Link  href="/create-review">
+						<p data-umami-event = "landlord page Submit a Review button" className="mt-2 inline-flex cursor-pointer items-center rounded-md border border-transparent bg-teal-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-teal-700 focus:outline-none focus:ring-2 focus:ring-teal-500 focus:ring-offset-2">
```
Comment (kellenwiltshire) [inline frontend/components/landlord/LandlordInfo.tsx:53]: Maybe we can create a different shorthand for the page name.

`landlord/${name}/submit-review` maybe. Have the other pages be similar.

**Review by kellenwiltshire — CHANGES_REQUESTED** [review]
Just a few changes to the implementation. I should have been clearer in the task.

File: frontend/components/home/hero.tsx (hunk 1 above)
Comment (stephenmartin2951) [reply frontend/components/home/hero.tsx:20]: umami is just the interface property name. It pulls through to the html as data-umami-event. I am working on fixing the errors caught by test-fe right now.

File: frontend/components/landlord/LandlordInfo.tsx (hunk 3 above)
Comment (stephenmartin2951) [reply frontend/components/landlord/LandlordInfo.tsx:53]: Sounds good. I am going to get all of the tests passing and then I can go back and change the naming conventions.

**Comment by stephenmartin2951** [comment]
I have to run, but I can change the naming convention later. 

Can you let me know how you would like the tags to look?

**Comment by kellenwiltshire** [comment]
> I have to run, but I can change the naming convention later. 
> 
> Can you let me know how you would like the tags to look? 

Sounds good!

I think if we went with `<page>/<button>` and for the navbar and footer then put that where the page would be

**Comment by stephenmartin2951** [comment]
Sounds good. I will get that done as soon as I can.

File: frontend/components/reviews/mobile-review-filters.tsx
Comment (stephenmartin2951) [inline frontend/components/reviews/mobile-review-filters.tsx:1]: Not sure why this was included (I definitely didn't indent them at any point, not sure what happened lol). Sorry about that.

File: frontend/components/reviews/ui/combobox.tsx
Comment (stephenmartin2951) [inline frontend/components/reviews/ui/combobox.tsx:1]: This too. Not sure what happened.

**Review by kellenwiltshire — APPROVED** [review]
Looks great!

**Pull request merged**